### PR TITLE
Privacy policy

### LINKED ISSUE #408: Add privacy policy page / popup
We don't necessarily need to have a popup, but with the addition of Google Analytics I'd like to be transparent as possible with our usage of their data. Basically, we'd outline that we don't share their data with Google / anyone else, what we use it for (hitcounts) and what data we actually look at. Also should probably tell them we're not connecting the analytics session to them logging in with Google (i.e. it's basically completely anonymized), as well as that 

We should probably double check all of these claims too, cause I wouldn't say I'm completely positive we're not inadvertently sharing their data with Google and etc

### PR DESCRIPTION
**Pull request by gannonprudhomme**
## Description

This adds the privacy policy to the landing page. This also adds an abstract `InfoDialog` component which both `AboutUs` and `PrivacyPolicy` use to remove the boilerplate/duplicate code from them.

Note that the method I used to center the links isn't very solid, but I know it's going to have to change anyways when Carlos does #496, so I think what I have now is probably fine.

## How to test

Read the privacy policy and make sure you agree with everything it says on it.

## Screenshots

|Landing Page|Privacy Policy|
|--|--|
|chrome_VPpn3rUM4k|chrome_5afGSIAul3|

## Related tasks

Closes #408

### CODE REVIEW

File: autoscheduler/frontend/src/components/LandingPage/LandingPage.css [hunk 1]
```diff
@@ -6,15 +6,13 @@
 }
 
 .dialog-container {
-    width: 100%;
+    padding-top: 32px;
```
Comment (ryan-conn) [inline autoscheduler/frontend/src/components/LandingPage/LandingPage.css:9]: Did you mean to add this in the latest commit? I don't remember us talking about changing this spacing, and both this and `dialog-link` have 32px top padding now which looks a bit odd to me. If it was intentional, you should only put the padding on one of the elements 

| Before | After |
|-|-|
| image | image |

**Review by ryan-conn — CHANGES_REQUESTED** [review]
Looks good except for a change from the latest commit that I don't think was intentional, see my comment

File: autoscheduler/frontend/src/components/LandingPage/LandingPage.css (hunk 1 above)
Comment (gannonprudhomme) [reply autoscheduler/frontend/src/components/LandingPage/LandingPage.css:9]: Correct yeah I definitely didn't meant to do that lol, will undo

**Review by ryan-conn — APPROVED** [review]

**Comment by gannonprudhomme** [comment]
I somehow missed that you approved this lmao

**Pull request merged**